install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+SOURCE_TAR_NAME=dp-cluster-reg.tar
+TARGET_FOLDER=dp-cluster-reg
+INSTALL_CMCLIENT=${CM:-false}
+
+if command -v python &>/dev/null
+then
+    echo "Python  is installed"
+else
+    echo "Python is not installed. Install Python an proceed with installation"
+    exit 1
+fi
+
+if [ "$CM" = true ]
+then
+    if [ "$VIRTUAL_ENV" != "" ]
+    then
+    echo "Installing cm_client in virtualenv $VIRTUAL_ENV"
+    elif [ "$VIRTUAL_ENV" == "" ]
+    then
+        echo "Installing cm_client in globl scope"
+    fi
+    pip install cm_client
+    status=$?
+    if [ $status -eq 0 ]
+    then
+        echo "Installation successful"
+    else
+        echo "Installation failed. resolve installation isuue and re-run the script"
```

Comment:
Typo: please change the message to: "Installation of cm_client module required for running the cluster registration script failed. Please install this manually and re-run the script"